error_code_report-0.1.0.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.exasol.com/error_code_report-0.1.0.json",
+  "type": "object",
+  "title": "Exasol error code report schema.",
+  "description": "This schema defines the structure of a report about error codes in an Exasol project.",
+  "additionalProperties": false,
+  "required": [
+    "projectName",
+    "projectVersion",
+    "errorsCodes"
+  ],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "description": "Reference to the schema. Setting this is required so that IDEs can assist you while writing your definition.",
```

Comment:
You are saying here this field is required but is not listed among the required fields. It might be confusing